services/121-service/test/payment/payment-in-progress.test.ts

```diff
@@ -1,59 +1,117 @@
 import { HttpStatus } from '@nestjs/common';
 
 import { Fsps } from '@121-service/src/fsps/enums/fsp-name.enum';
+import { PaymentEventDataDto } from '@121-service/src/payments/payment-events/dtos/payment-event-data.dto';
+import { PaymentEvent } from '@121-service/src/payments/payment-events/enums/payment-event.enum';
+import { TransactionStatusEnum } from '@121-service/src/payments/transactions/enums/transaction-status.enum';
+import { TransactionEventInterface } from '@121-service/src/payments/transactions/transaction-events/dto/transaction-event-data.dto';
+import { TransactionEventType } from '@121-service/src/payments/transactions/transaction-events/enum/transaction-event-type.enum';
 import { SeedScript } from '@121-service/src/scripts/enum/seed-script.enum';
-import { registrationsPV } from '@121-service/test/fixtures/scoped-registrations';
+import {
+  registrationNotScopedPv,
+  registrationScopedTurkanaNorthPv,
+} from '@121-service/test/fixtures/scoped-registrations';
 import {
   createAndStartPayment,
+  createPayment,
+  getPaymentEvents,
+  getPayments,
   getProgramPaymentsStatus,
+  getTransactions,
   retryPayment,
+  startPayment,
   waitForPaymentTransactionsToComplete,
 } from '@121-service/test/helpers/program.helper';
-import { seedIncludedRegistrations } from '@121-service/test/helpers/registration.helper';
+import {
+  doPaymentAndWaitForCompletion,
+  getTransactionEvents,
+  seedIncludedRegistrations,
+  updateRegistration,
+} from '@121-service/test/helpers/registration.helper';
 import {
   getAccessToken,
   resetDB,
 } from '@121-service/test/helpers/utility.helper';
 import {
   programIdOCW,
   programIdPV,
-  registrationOCW4,
   registrationsOCW,
 } from '@121-service/test/registrations/pagination/pagination-data';
 
-// NOTE: these test focus on payment being in progress because of active queue. Being in-progress because of actions is too short to cover with API-tests (with low volumes) and is either way better already covered by unit test on payments-creation.service.spec.ts
+// NOTE: these test focus on payment being in progress because of active queue or payment lock. Being in-progress because of actions is too short to cover with API-tests (with low volumes) and is either way better already covered by unit test on payments-creation.service.spec.ts
```

Comment:
this comment still mentions actions so seems out of date. Because of your change of focus, the whole comment can be removed I think.